### DISCUSSION TITLE
Hitting Walls While Installing Crouton

### DISCUSSION THREAD

**Discussion by nebcanuck** [dnschneid/crouton#5011] [Q&A]
Hi there,

I recently acquired a Chromebook and have been attempting to install Crouton. The first issue I ran into is that sudo is now disabled within the crosh shell. I have not found any way to run sudo commands there, which means that I don't believe I can use the terminal launched by pressing Ctrl+Alt+T to install Crouton.

Because of that, I tried installing Crouton using the Virtual Terminal by pressing Ctrl+Alt+Refresh. I had some difficulty with this because I ran into an error that read "sed: can't read common/cras_messages.h: no such file or directory". I found a discussion on this forum indicating that fixing this requires using CROUTON_BRANCH=silence, which will disable sound in the chroot. I did this and it worked - the chroot was installed successfully.

Unfortunately, when I tried launching the chroot using sudo startxfce4, I received another error that said "xorg x11 servers cannot be run from frecon", and that I should try launching the chroot from inside the crosh terminal. Since the command to launch the chroot requires sudo privileges, I end up back where I started, unable to perform the required command.

I assume that I'm missing something, but I can't find a solution in the Readme, any forums I've searched, or the discussion page here. Please let me know if there's something I can do to fix this!

**Comment by dnschneid** [comment]
Well, if it makes you feel better, the fact that you can use sudo from VT2 is an improvement :)

I had some ideas about how to launch graphical sessions [in this thread](https://github.com/dnschneid/crouton/discussions/5005#discussioncomment-6855438) but I don't know if anyone tried them.

**Reply by nebcanuck** [reply]
Thank you for the quick response and the link. I will probably see if I can figure out how to pull off some of your proposals, but it's not pressing, since it's primarily a game related issue that I'm trying to sort out.

**Comment by CroutonIsFun** [comment]
[How to install and run crouton](https://github.com/dnschneid/crouton/discussions/5013)
